LICENSE

```diff
@@ -199,3 +199,21 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+=======================================================================
+Apache SkyWalking Eyes Subcomponents:
+
+The Apache SkyWalking Eyes project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
+========================================================================
+MIT licenses
+========================================================================
+
+The following components are provided under the MIT license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+
+    languages.yaml from GitHub: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml MIT
```

Comment:
@wu-sheng please recheck this, I copy this file from GitHub's linguist repo

Reply:
From the license's perspective, I think it is fine. About the paper work, I think you miss `licenses/LICENSE-linguist.txt`?

Reply:
Is this related to the runtime? Could you explain a little about how we use this?

Reply:
This file contains a full list of mappings between file types and their extensions, we parse this at runtime to get the mapping and map the extensions to our comment styles.

This could be downloaded at runtime, but that would be troublesome for people who have poor/no network access

Reply:
Containing this file in our binary makes it more self-contained IMO

Reply:
> This file contains a full list of mappings between file types and their extensions, we parse this at runtime to get the mapping and map the extensions to our comment styles.

Then this should be included in source codes license and dist license.

> This could be downloaded at runtime, but that would be troublesome for people who have poor/no network access

No need, I was just checking from license perspective.